pyexasol/http_transport.py

```diff
@@ -299,6 +334,24 @@ def export_to_callback(self, callback, dst, callback_params=None):
             raise e
 
     def import_from_callback(self, callback, src, callback_params=None):
+        """
+        Import chunk of data using callback function. 
+
+        Args:
+            callback:
+                Callback function.
```

Comment:
Describe signature of callback function

Reply:
Good point; this is a future task (improve the documentation). With this PR, I mainly want to migrate and ensure the basics work and are available, so future improvements will be simple and straightforward.